packages/docs/content/5_components/alert.mdx

```diff
@@ -55,10 +101,6 @@ The following Sass variables can be overridden to theme alerts:
 - `$alert-link-color-active` - Used to control the alert link text color when active
 - `$alert-link-color-focus` - Used to control the alert link text color when focused
 
-### Learn more
-
-- [18F Content Guide \- Active voice](https://content-guide.18f.gov/active-voice/)
-
 ## Google Analytics
 
 **Analytics event tracking is disabled by default.**
```

Comment:
I can't comment on the actual line because it's not part of the diff, but the JS syntax highlighting doesn't seem to work. I tried lowercasing it and simplifying it to `js`, but it still doesn't seem to work for me. There are other pages in your site where `css` and `html` blocks have working syntax highlighting 😕